src/test.zig

```diff
@@ -1860,7 +1860,8 @@ pub const TestContext = struct {
 
                             .wasmtime => |wasmtime_bin_name| if (enable_wasmtime) {
                                 try argv.append(wasmtime_bin_name);
-                                try argv.append("--dir=.");
+                                try argv.append("--mapdir=/::.");
+                                try argv.append("--mapdir=/tmp::/tmp");
```

Comment:
I assume this was changed to add those preopens for the test cases that were added. Would it be possible to emulate the preopens instead?

Reply:
what do you mean? can you point me to an example?

Reply:
to be clear: what I meant here is I am not sure how you would emulate the preopens; as far as I've understood the tests are run through wasmtime so you need to pass at least two mappings explicitly to exercise the code path. 

I don't know if there is an example of emulating preopens already somewhere in the codebase, in that case I will just copy whatever it's done there